packages/packem/src/rollup/plugins/css/types/sass.d.ts

```
@@ -0,0 +1,49 @@
+declare namespace sass {
+    type Data = Error | { contents: string } | { file: string } | null;
+
+    // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
+    type Importer = (url: string, previous: string, done: (data: Data) => void) => Data | void;
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)